### ISSUE TITLE
Rename to SKON-atom

### ISSUE THREAD

**Issue by Tranberry** [SpaceKrakens/atom-SKON#1]
And do the same to all SKON highlighting and addon stuff, that way https://github.com/SpaceKrakens will look more ordered. What do you think?

**Comment by NogginBops** [comment]
I think we should move all the highlighting in one single repo with different subfolders. That way it's easier to sync changes and make highlighting consistent between editors.

**Comment by Tranberry** [comment]
yes I agree, that's even better. I thought there was some superb reason why it was not so.
